packages/local_auth/lib/local_auth.dart

```diff
@@ -66,7 +66,7 @@ class LocalAuthentication {
   /// authentication (e.g. lack of relevant hardware). This might throw
   /// [PlatformException] with error code [otherOperatingSystem] on the iOS
   /// simulator.
-  Future<bool> authenticateWithBiometrics({
+  Future<bool?> authenticateWithBiometrics({
```

Comment:
This should not return a nullable type.

Reply:
Updated to return `Future<bool>` instead of `Future<bool?>`